### PR TITLE
DOC: stats: Added tutorial documentation for the generalized normal distribution. #7171

### PR DESCRIPTION
**Pull request by DominicAntonacci**
This is my first pull request, so let me know if there's anything that needs to be improved/fixed.

### CODE REVIEW

File: doc/source/tutorial/stats/continuous_gennorm.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Moments
+-------
+
+.. math::
+    \begin{eqnarray*}
+    \mu & = & 0 \\
+    m_{n} & = & 0 \\
+    m_{d} & = & 0 \\
+    \mu_2 & = &  \mu_2  =  \frac{\Gamma\left(3/\beta\right)}{\gamma\left(1/\beta\right)} \\
```
Comment (WarrenWeckesser) [inline doc/source/tutorial/stats/continuous_gennorm.rst]: You have `\mu_2` twice.  This should be just `\mu_2 & = & \frac{\Gamma ...`

**Review by WarrenWeckesser — CHANGES_REQUESTED** [review]
This is great, thanks!  I requested one change in-line.

The only other suggestion is that you follow the conventions for commit messages used by scipy and numpy described in https://docs.scipy.org/doc/numpy/dev/gitwash/development_workflow.html#writing-the-commit-message (admittedly, that can be hard to find for a new contributor).  Specifically, commit messages that are updates to the documentation (e.g. docstrings and tutorials) should have the prefix `DOC:`, just like the heading of this pull request.

File: doc/source/tutorial/stats/continuous_gennorm.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+a single shape parameter :math:`\beta>0`. It reduces to a number of common
+distributions.
+
+Functions
+---------
+
+.. math::
+    :nowrap:
+
+    \begin{eqnarray*} f\left(x\right) & = &\frac{\beta}{2\Gamma(1/\beta)} e^{-\left|x\right|^{\beta}} \end{eqnarray*}
```
Comment (WarrenWeckesser) [inline doc/source/tutorial/stats/continuous_gennorm.rst]: Here and elsewhere: the convention for specifying the funtion arguments is to include the parameter, separated from x with a semicolon.  E.g. `f\left(x; \beta \right) & = & ...`.

**Comment by WarrenWeckesser** [comment]
I just commented in-line about one more detail that should be fixed.

**Comment by DominicAntonacci** [comment]
Sorry about the delay on that! Should be ready to go.

**Comment by WarrenWeckesser** [comment]
Thanks, Dominic.  Merged.

**Pull request merged**